go.mod

```diff
@@ -66,11 +66,11 @@ require (
 
 // Automated release process replaces
 // DO NOT EDIT: CHANGES TO THE 3 BELOW LINES WILL BREAK AUTOMATED RELEASES
-replace github.com/cyberark/conjur-api-go => github.com/cyberark/conjur-api-go latest
+replace github.com/cyberark/conjur-api-go => github.com/cyberark/conjur-api-go v0.10.1
```

Comment:
This won't break the automated releases?

Reply:
Yep. it will. Thought the build would fail for that. Hmmm. I'll update. Thanks, @rpothier!